### DISCUSSION TITLE
Pesquisa sobre tooling no desenvolvimento front end

### DISCUSSION THREAD

**Discussion by woliveiras** [frontendbr/forum#1154] [Discussão]
Fala, galera!

Estou fazendo uma pesquisa sobre o ferramental no desenvolvimento front end.

É um formulário grande, mas da para responder rapidinho, pois são perguntas bem fechadas. :) 

Respondam por favor e me ajudem a divulgar nas redes. ❤️ 

https://goo.gl/forms/YLl08W5Kd1ZnQ6zg2

**Comment by matheusrod92** [comment]
@woliveiras achei muito boa a pesquisa e gostaria de saber se você vai disponibilizar os resultados em alguma rede social sua

**Comment by woliveiras** [comment]
Vou sim, @matheusrod92! :D 

Vou postar no Medium: https://medium.com/@woliveiras

E depois compartilhar em tudo que é canto desse universo (inclusive aqui na issue :)~)

**Comment by drianoaz** [comment]
@woliveiras fiquei assustado com a quantidade de tecnologias que nunca ouvi falar, mas respondi sua pergunta, espero ter ajudado e quero ver os resultados!

**Comment by woliveiras** [comment]
Valeu, @drianoaz! 

Se tudo der certo vou conseguir uns cursos pra galera sobre essas tecnologias! :D

**Comment by klarkc** [comment]
Curioso sobre como anda o isomorfismo nas aplicações
